test/test_samples/behave/behave_runner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I don't see any tests that check running behave as a script, so I assume these 2 scripts can be safely removed?

Reply:
I took the pytest directory as a template, that spawned their existence. Currently they show how to call behave, which may be handy. Maybe I had to add a test to justify their presence :-)

Reply:
It's used in test\tests\pytestScript.test.ts. If you're not planning to replicate similar tests, I would suggest removing these scripts. Up to you 🙂

Reply:
I will add a test.